{{ cookiecutter.repo_name }}/docs/contributor_guide/pre_commit_hooks.md

```diff
@@ -93,17 +94,34 @@ updating and auditing the `.secrets.baseline` file.
 
 ## Keeping specific Jupyter notebook outputs
 
+>  ℹ️ Currently (March 2020) there is no way to add tags and/or metadata to Google Colab notebooks. It's strongly
+> suggested that you download the Colab as a .ipynb file, and edit tags and/or metadata using Jupyter _before_
+> committing the code if you want to keep some outputs.
+
```

Comment:
Just a thought but it is a bit weird to find information about Google Colab as first thing first in a section about Jupyter notebook outputs.

I'd move it at the end of this section.

Reply:
Resolved in commit [`6164ea5`](https://github.com/ukgovdatascience/govcookiecutter/pull/24/commits/6164ea5f1b6a3d1f0887ae7e20a8aff08b2b9a7e).